src/utils.ts

```diff
@@ -31,7 +31,23 @@ export function watchEffectOnce<T>(watcher: () => T, fn: Function) {
  */
 export function bindPluginMethods(plugin: any, exclude: string[]) {
   Object.getOwnPropertyNames(Object.getPrototypeOf(plugin))
-    .filter(method => !exclude.includes(method)) 
+    .filter(method => !exclude.includes(method))
     // eslint-disable-next-line security/detect-object-injection
-    .forEach(method => (plugin[method] = plugin[method].bind(plugin))); 
+    .forEach(method => (plugin[method] = plugin[method].bind(plugin)));
+}
+
+/**
+ * @ignore
+ * Helper function to map the v1 `redirect_uri` option to the v2 `authorizationParams.redirect_uri`
+ * and log a warning.
+ */
+export function deprecateRedirectUri(options?: any) {
```

Comment:
Should we export it ?

Reply:
I don't think this is available for a user of `auth0-vue` to import as we don't expose `src/utils.ts` to them only the exports of `src/index.ts`. It's exported from this file os it can be used in `plugin.ts` (I figured it fit better in here)

Reply:
Oh, my bad. For some reason I missed this was a different file (utis), sorry!